### PR TITLE
ISPN-8868 exposes JMX methods

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-8868

* force release
* remove
* isDefined
* isLocked

### CODE REVIEW

File: lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java [hunk 1]
```diff
@@ -93,6 +94,10 @@ private static void registerClusteredLockManager(GlobalComponentRegistry registr
          if (counterManager == null || !(counterManager instanceof EmbeddedClusteredLockManager)) {
```
Comment (pruivo) [inline lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java]: `counterManager`?

Comment (karesti) [reply lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java]: this is copy/paste

File: lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java [hunk 2]
```diff
@@ -96,14 +118,25 @@ public ClusteredLockConfiguration getConfiguration(String name) {
...
+      if (trace) {
+         log.tracef("LOCK[%s] isDefined has been called", name);
+      }
       if (cache == null) {
          cache = extractCacheHolder(cacheHolderFuture).getClusteredLockCache();
       }
       return cache.containsKey(new ClusteredLockKey(ByteString.fromString(name)));
    }
 
+   @ManagedOperation(
```
Comment (pruivo) [inline lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java]: you can't reuse this methods. 
`CompletableFuture` isn't serializable and the operation will fail (in JConsole at least) with `NotSerializableException`

**Comment by pruivo** [comment]
since you are exposing some operation, don't you want to expose the remaining one:
* is-locked
* configuration
* get lock names (?)

**Comment by karesti** [comment]
@pruivo yeah, maybe. I did the minimum asked but why not

File: lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java (hunk 2 above)
Comment (karesti) [reply lock/src/main/java/org/infinispan/lock/impl/manager/EmbeddedClusteredLockManager.java]: ok, I need to make them sync ?

**Comment by karesti** [comment]
@pruivo I did the modifications and added a two additional methods that might be useful

**Comment by karesti** [comment]
@pruivo test failures unrelated ...

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**